package tastetrail_backend.repository;

import java.util.Optional;

import org.springframework.data.jpa.repository.JpaRepository;

import tastetrail_backend.entity.User;

public interface UserRepository  extends JpaRepository<User, Long>{
	Optional<User> findByUsername(String username);
	
	Optional<User> findByEmail(String email);
}
